core/cryptography/build.gradle.kts

```diff
@@ -105,7 +99,7 @@ kotlin {
     }
 }
 
-project.appleTargets().forEach {
+listOf("iosArm64", "iosSimulatorArm64", "macosArm64").forEach {
```

Comment:
Or you delete the targets x86 targets from https://github.com/wireapp/kalium/blob/26ed8afdd26a470ef8162696e13905c8e9fa8fcd/buildSrc/src/main/kotlin/com/wire/kalium/plugins/CommonDarwinConfig.kt#L25

Reply:
ahh make sense did not know about this config